src/assets/scss/ProfileDisplay.scss

```diff
@@ -1,6 +1,9 @@
+@import '../../assets/scss/_vars'; 
+
 .profileDisplay {
   position: relative;
   padding: 0 2.4rem;
+  color: $black;
```

Comment:
Just as a note, if we do this, we need to standardize across the board meaning all colors will be held as sass variables

Reply:
Yes.